packages/react/src/ReactServer.fb.js

```diff
@@ -7,6 +7,8 @@
  * @flow
  */
 
+import {enableViewTransition} from 'shared/ReactFeatureFlags';
```

Comment:
To be safe, can you remove the feature flag and just continue exporting it as `unstable_ViewTransition`? I have no idea if turning this off would break our infra.